apps/antalmanac/src/components/Calendar/toolbar/ScheduleSelect/ScheduleSelect.tsx

```diff
@@ -116,17 +118,23 @@ export function SelectSchedulePopover(props: { scheduleNames: string[] }) {
                                         justifyContent: 'flex-start',
                                         background:
                                             index === currentScheduleIndex ? theme.palette.action.selected : undefined,
+                                        '&:hover': {
+                                            backgroundColor: theme.palette.action.hover,
+                                            transition: 'background-color 0.2s',
+                                        },
```

Comment:
question: why are we adding this hover transition? If this is an arbitrary preference, it would be better to omit this from the PR, as AntAlmanac doesn't much use transitions